### ISSUE TITLE
Helping with French translation

### ISSUE THREAD

**Issue by Marc66** [aonez/Keka#670]
I would like to help with French translation as I see it is quite outdated but I don't know how to proceed

I cloned the Keka repository and can edit files there but I'm not sure how I can send them as pull requests or commits. In fact, I'm not even sure I need to clone the repository.

Can you explain me the easiest way to proceed? Do I need to install a git client on my Mac or can I do everything on-line?

Thank you

**Comment by aonez** [comment]
Hi @Marc66! Thanks for the willing 😊

You can do it all on-line:

- Clone the repository (already did)
- Edit all the files you need to edit. You can do it with the Github editor, simply open the file and click in the ✏️
- Once you've edited all the files, go to pull requests -> new pull request. Usually you don't need to touch anything here, just create the pull.

Let me know if you need anything during the process. And thanks a lot in advance!

**Comment by Marc66** [comment]
> Hi @Marc66! Thanks for the willing 😊
> 
> Let me know if you need anything during the process. And thanks a lot in advance!

I'm done but I noticed someone else started to update as well. I see he only touched 2 files and left him a message to continue after my commits are merged in your master branch.

BTW, how do I synchronise my fork with your master so I can continue updating in the future?

**Comment by aonez** [comment]
To sync your fork simply do a pull request, click on the “compare across forks” link and select your fork as the base repository and aonez/Keka as the head repository. Be sure you have “master” selected as the branch for booth sides.

That will create a pull in your fork and once it is merged you’ll be synced. Let me know it that works for you 😊

**Comment by aonez** [comment]
Here a build with your translation: [Keka-French.7z](https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-French.7z)

**Comment by Marc66** [comment]
> Here a build with your translation: [Keka-French.7z](https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-French.7z)

Translation is fine except for one string I missed. Created a new Pull request
I went overt the entire fr.lproj folder of this binary to make sure everything is fine

**Comment by aonez** [comment]
@Marc66 your translation is up and running! Thanks a lot for that 😊
